languages/java/oso/pom.xml

```diff
@@ -97,7 +97,7 @@
       <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-javadoc-plugin</artifactId>
-          <version>2.9.1</version>
+          <version>3.0.1</version>
```

Comment:
I don't think we can upgrade this without testing that it's compatible with the maven central stuff.